### PR TITLE
feat: set airVersion and goVersion with ldflags in release process

### LINKED ISSUE #105: air version is not match
image
image

### PR DESCRIPTION
**Pull request by tkms0106**
resolves #105 

release workflow test
https://github.com/tkms0106/air/runs/1065558109?check_suite_focus=true

### CODE REVIEW

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -51,6 +51,11 @@ jobs:
 
       - name: Setup Go
         uses: actions/setup-go@v2
+        with:
+          go-version: 1.14
+
+      - name: set GOVERSION
+        run: echo "::set-env name=GOVERSION::$(go version | sed -r 's/go version go(.*)\ .*/\1/')"
```
Comment (tkms0106) [inline .github/workflows/go.yml:58]: https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable

File: .goreleaser.yml [hunk 2]
```diff
@@ -6,6 +6,8 @@ builds:
     ignore:
       - goos: darwin
         goarch: 386
+    ldflags:
+      - -X main.airVersion={{.Version}} -X main.goVersion={{.Env.GOVERSION}}
```
Comment (tkms0106) [inline .goreleaser.yml:10]: https://goreleaser.com/customization/build/
https://goreleaser.com/customization/templates/

File: .github/workflows/go.yml (hunk 1 above)
Comment (tkms0106) [reply .github/workflows/go.yml:58]: e.g. go1.14.8
image

**Comment by xiantang** [comment]
i will check it soon

**Pull request merged**